ivy/functional/frontends/torch/loss_functions.py

```diff
@@ -100,4 +100,17 @@ def binary_cross_entropy(
     if weight is not None:
         result = ivy.multiply(weight, result)
     result = reduction(result)
-    return result
+    return 0
+
+
+def nll_loss(
+        input, target, weight=None, size_average=None, ignore_index=- 100, reduce=None, reduction='mean'
+):
+    input = ivy.exp(input)
+    depth = ivy.max(target) + 1
+    one_hot_target = ivy.one_hot(target, depth)
```

Comment:
I added one-hot encoding because torch.nll accepts indeces as targets. I think it should be done in torch frontend cross_entropy for the same reason